packages/gestalt/src/Button.css

```diff
@@ -10,17 +10,20 @@
 
 .sm {
   composes: small from "./Layout.css";
-  padding: 6px 14px;
+  composes: paddingX3 from "./boxWhitespace.css";
+  composes: paddingY1 from "./boxWhitespace.css";
```

Comment:
Since we have a `min-height` set on each button, the height doesn't change when we update the vertical padding. This approach ultimately results in less CSS being put out.